tools/setup_perf_cluster.sh

```diff
@@ -473,6 +494,9 @@ function check_image_versions() {
 if [[ $SOURCED == 0 ]]; then
   # Normal mode: all at once:
   update_gcp_opts
+  if [ "${SKIP_CREATE}" != "true" ]; then
+    create_vm_cluster_all
+  fi
   setup_all
 
```

Comment:
I haven't used the script as is in a long time, I only "source" it and run functions 1 by 1

I think "run_tests" probably doesn't work the first time as it takes a while to get the ips for ingress for a brand new cluster, so in theory we would need a "wait_for_ready" before "run_tests" below

I think also in automated way, you probably don't need to start "fortio report &"

Would you consider sourcing the file and calling the functions individually maybe ?

Reply:
It will be another wrapper script, I can give it a go